src/index.js

```diff
@@ -25,6 +26,9 @@ $('.board').click(function(e){
   // game.currentPlayers[game.currentPlayerIndex]
 });
 
+quitBtn.click(function(e) {
```

Comment:
I don't think this would need an e as we're not passing an event to prevent the default. Have you tried running the code without it?